docs/index.bs

```diff
@@ -1986,78 +1983,64 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       <dfn method for="Cache"><code>add(|request|)</code></dfn> method *must* run these steps:
 
         1. Let |requests| be an array containing only |request|.
-        1. Set |responseArrayPromise| to the result of running the algorithm specified in {{Cache/addAll(requests)}} passing |requests| as the argument.
-        1. Return the result of <a>transforming</a> |responseArrayPromise| with a fulfillment handler that returns undefined.
+        1. Let |responseArrayPromise| be the result of running the algorithm specified in {{Cache/addAll(requests)}} passing |requests| as the argument.
+        1. Return the result of [=transforming=] |responseArrayPromise| with a fulfillment handler that returns undefined.
     </section>
 
     <section algorithm="cache-addAll">
       <h4 id="cache-addAll">{{Cache/addAll(requests)}}</h4>
 
       <dfn method for="Cache"><code>addAll(|requests|)</code></dfn> method *must* run these steps:
 
-        1. Let |responsePromiseArray| be an empty array.
-        1. Let |requestArray| be an empty array.
+        1. Let |r| be null.
+        1. Let |responsePromises| be an empty [=list=].
+        1. Let |requestList| be an empty [=list=].
         1. For each |request| whose type is {{Request}} in |requests|:
-            1. Let |r| be |request|'s [=Request/request=].
+            1. Set |r| to |request|'s [=Request/request=].
             1. If |r|'s [=request/url=]'s [=url/scheme=] is not one of "<code>http</code>" and "<code>https</code>", or |r|'s [=request/method=] is not \`<code>GET</code>\`, return a <a>promise</a> rejected with a <code>TypeError</code>.
         1. For each |request| in |requests|:
-            1. Let |r| be the associated [=Request/request=] of the result of invoking the initial value of {{Request}} as constructor with |request| as its argument. If this <a>throws</a> an exception, return a <a>promise</a> rejected with that exception.
+            1. Set |r| to the associated [=Request/request=] of the result of invoking the initial value of {{Request}} as constructor with |request| as its argument. If this <a>throws</a> an exception, return [=a promise rejected with=] that exception.
             1. If |r|'s [=request/url=]'s [=url/scheme=] is not one of "<code>http</code>" and "<code>https</code>", then:
-                1. [=fetch/Terminate=] all the ongoing <a>fetches</a> initiated by |requests| with reason *fatal*.
+                1. [=fetch/Terminate=] all the ongoing [=fetches=] initiated by |requests| with reason *fatal*.
```

Comment:
We've since dropped the "reason". There's an aborted flag which should probably be set in this case. https://fetch.spec.whatwg.org/#concept-fetch-terminate

Reply:
Addressed.